src/site/stages/build/process-cms-exports/schemas/transformed/node-support_service.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  type: 'object',
+  properties: {
+    contentModelType: { enum: ['node-support_service'] },
+    entity: {
+      type: 'object',
+      properties: {
+        entityType: { enum: ['node'] },
+        entityBundle: { enum: ['support_service'] },
+        title: { type: 'string' },
+        fieldLink: {
+          type: ['object', 'null'],
+          properties: {
+            url: {
+              type: 'object',
+              properties: {
+                path: { type: 'string' },
+              },
+              required: ['path'],
+            },
```

Comment:
Man, I wish we could refactor these while we're here. Glad this makes it easy for somebody else to do later, though.

Wishful thinking. :smile: